### PR TITLE
Feature/map

### PR DESCRIPTION
**Pull request by GiftLanga**
added map component on the homepage

### CODE REVIEW

File: waterlog/src/Pages/Home.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-      <div>
-        <p className="home-text">Alerts Map</p>
-      <AlertsMap />
-      <Wastage />
-      <p className="home-text">Tank Levels</p>
-      <TankLevels />
-      </div>
-    );
-  }
+	render() {
```
Comment (RetroGuillaume) [inline waterlog/src/Pages/Home.js]: What up with the spacing ? Does not look like 4 spaces

File: waterlog/src/Components/Map.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	{ lat: -25.782454, lon: 28.337993 },
+	{ lat: -25.782456, lon: 28.337866 },
+    { lat: -25.782634, lon: 28.338039 },
+    { lat: -25.783408, lon: 28.336180}
+];
+
+export default class MapComponent extends Component {
+	constructor() {
+		super();
+		this.state = {
```
Comment (laurent-mbuyu) [inline waterlog/src/Components/Map.js:16]: You might as well initialize your state with the markers array of object at the third position

Comment (GiftLanga) [reply waterlog/src/Components/Map.js:16]: I created the markers array in the state and initialized it

File: waterlog/src/Pages/Home.js (hunk 1 above)
Comment (GiftLanga) [reply waterlog/src/Pages/Home.js]: we are using 2 spaces

Comment (GiftLanga) [reply waterlog/src/Pages/Home.js]: Someone recommended that we use 2 spaces instead of 4

File: waterlog/src/Components/AlertsMap.js [hunk 3]
```diff
@@ -1,11 +1,21 @@
...
 const AlertsMap = (props) => {
-    return (
-      <div className="alerts-map">
-        <Loader />
-      </div>
-    )
-}
-export default AlertsMap;
\ No newline at end of file
+	const { error, loading, sensorLocations } = props;
```
Comment (Tebogo-Seshibe-Retro) [inline waterlog/src/Components/AlertsMap.js]: You can place this inside the function parameter list.
```js
const AlertsMap = ({ error, loading, sensorLocations }) => {
// function body here
};
```

Comment (GiftLanga) [reply waterlog/src/Components/AlertsMap.js]: I placed it in the function parameter list.

**Review by laurent-mbuyu — APPROVED** [review]

File: waterlog/src/icons/MapIcons.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+	popupAnchor: null,
+	shadowUrl: null,
+	shadowSize: null,
+	shadowAnchor: null,
+	iconSize: new L.Point(60, 75)
+});
+
+export const sensorFaultIcon = new L.icon({
+	iconUrl: require('../images/redCircleFaultMarker.png'),
+	//iconRetinaUrl: require('../img/marker-pin-person.svg'),
```
Comment (Tebogo-Seshibe-Retro) [inline waterlog/src/icons/MapIcons.js]: Please remove this line.

**Review by Tebogo-Seshibe-Retro — CHANGES_REQUESTED** [review]

File: waterlog/src/icons/MapIcons.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+	popupAnchor: null,
+	shadowUrl: null,
+	shadowSize: null,
+	shadowAnchor: null,
+	iconSize: new L.Point(60, 75)
+});
+
+export const sensorRedIcon = new L.icon({
+	iconUrl: require('../images/redCircleMarker.png'),
+	//iconRetinaUrl: require('../img/marker-pin-person.svg'),
```
Comment (Tebogo-Seshibe-Retro) [inline waterlog/src/icons/MapIcons.js]: This too.

**Review by Tebogo-Seshibe-Retro — CHANGES_REQUESTED** [review]

File: waterlog/src/icons/MapIcons.js (hunk 5 above)
Comment (GiftLanga) [reply waterlog/src/icons/MapIcons.js]: I removed the comments

**Review by Tebogo-Seshibe-Retro — APPROVED** [review]

**Review by ash-rabbit — APPROVED** [review]

**Review by CeeyarRbt — APPROVED** [review]

**Pull request merged**